Makefile

```diff
@@ -53,11 +61,23 @@ debug-dockerignore:
 
 .PHONY: run-virtual
 run-virtual: build-virtual
-	$(CCF_PREFIX)/bin/sandbox.sh -p $(BUILD)/liblskv.virtual.so --http2
+	VENV_DIR=.venv $(CCF_PREFIX)/bin/sandbox.sh -p $(BUILD)/liblskv.virtual.so --http2
+
+.PHONY: run-virtual-unsafe
```

Comment:
For 3.0.0, we will only publish an unsafe package for the sgx target.

Reply:
Will there be a way to get the verbose CCF logs in virtual mode?

Reply:
That's a good point. We could simply add a runtime parameter for the enclave logging in virtual mode (it is a build-time parameter in SGX as logging level should change enclave measurement for security reasons).

Reply:
https://github.com/microsoft/CCF/issues/4441